web/cypress/integration/consumer/create_and_delete_consumer.spec.js

```diff
@@ -25,10 +25,9 @@ context('Create and Delete Consumer', () => {
   });
 
   it('creates consumer with key-auth', function () {
-    cy.visit('/');
-    cy.contains('Consumer').click();
+    cy.visit('/consumer/list');
```

Comment:
This will not cover whether the left menu button can jump to the corresponding page.